src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java

```diff
@@ -9,15 +9,33 @@
 import org.broadinstitute.hellbender.engine.ReferenceContext;
 import org.broadinstitute.hellbender.engine.VariantWalker;
 
+/**
+ *
+ * Count variants in a VCF file.
+ *
+ * <h3> Input </h3>
+ * <ul>
+ *     <li> A single VCF file. </li>
+ * </ul>
+ *
+ * <h3> Example: </h3>
+ * <pre>
+ *  gatk CountVariants \
+ *      -V input_variants.vcf
+ * </pre>
+ */
 @DocumentedFeature
 @CommandLineProgramProperties(
-        summary = "Walks over the input data set, calculating the number of variants seen.",
-        oneLineSummary = "Count variants in a VCF file",
+        summary = CountVariants.USAGE_SUMMARY,
+        oneLineSummary = CountVariants.USAGE_ONE_LINE_SUMMARY,
         programGroup = VariantProgramGroup.class
 )
 public final class CountVariants extends VariantWalker{
     private long count = 0;
 
+    static final String USAGE_ONE_LINE_SUMMARY = "Count variants in a VCF file";
+    static final String USAGE_SUMMARY = "Walks over the input data set, calculating the number of variants seen.";
```

Comment:
Walks over the input data set, calculating the number of variants seen. --> Counts variant records in a VCF file, regardless of filter status.